lib/content-services/src/lib/aspect-list/aspect-list.component.spec.ts

```diff
@@ -85,6 +85,21 @@ const customAspectListMock: AspectEntry[] = [{
             }
         ]
     }
+},
+{
+    entry: {
+        parentId: 'cst:nonamedAspect',
+        id: 'cst:nonamedAspect',
```

Comment:
can we make the id different from the parentId? If we accidentally changed in the html the title to be the parentId, the tests will not fail